src/Controller/ResultController.php

```diff
@@ -27,4 +38,19 @@ public function getResults(Request $request, array $requestAttributes): Response
 
         return $this->renderer->renderView('admin-results-listing.phtml', ['data' => $data]);
```

Comment:
Passing the array of rows obtained directly from a select statement from ORM->Controller->View is totally defeating the purpose of the ORM and couples your view to the names of the columns in your application's tables.
Results data should be encapsulated/wrapped into some view model objects and those be passed to the view. As a rule, you shouldn't have array dereferencing in views (that is taking data from arrays by associative keys in order to print it out in the view). Data should be passed to views in objects with getters easy to call and use from view templates.